book/version_control.md

```diff
@@ -246,107 +241,44 @@ sandpit/
-The second will ignore all files within a folder named `sandpit/`.
-Note that if there are multiple folders in the project named `sandpit/`, they will all be excluded.
-You should make the pattern more specific if you want to exclude particular directories.
-The third pattern will exclude all `.csv` files, regardless of which folder or sub-folder they are in.
+The first pattern tells Git to ignore any file with the exact name `secrets.yaml`. The second will ignore all files within a folder named `sandpit/`. Note that if there are multiple folders in the project named `sandpit/`, they will all be excluded. You should make the pattern more specific if you want to exclude particular directories. The third pattern will exclude all `.csv` files, regardless of which folder or sub-folder they are in.
 
 ```{caution}
-Files that are already being tracked by Git will not be excluded when you add new patterns to your `.gitignore` file.
-Ensure that you set up your `.gitignore` before adding files to be tracked in your repository.
+Files that are already being tracked by Git will not be excluded when you add new patterns to your `.gitignore` file. Ensure that you set up your `.gitignore` before adding files to be tracked in your repository.
 ```
 
 Please see the [Git documentation for `.gitignore`](https://git-scm.com/docs/gitignore) for more details.
 
-While we may not want to share files containing credentials, configuration and data, it can be useful to provide dummy examples of these files.
-Demonstrating the format of these files can help others to reuse your code, without sharing sensitive information.
-
-
-#### Using environmental variables
-
-If your code depends on credentials of some kind, these should not be explicitly written in your code.
-They can be stored in configuration files, which should be excluded from version control, or better still in local environment variables.
-
-Environment variables are variables which are available in a particular environment.
-In this context, our environment is the user environment that we are running our code from.
-
-In Unix systems (e.g. Linux and Mac), environment variables can be set in the terminal using `export` and deleted using `unset`:
-
-```
-export SECRET_KEY="mysupersecretpassword"
-unset SECRET_KEY
-```
-
-In Windows, the equivalent to this is:
-
-```
-setx SECRET_KEY "mysupersecretpassword"
-reg delete HKCU\Environment /F /V SECRET_KEY
-```
-
-Once stored in environmental variables, these variables will remain available on your machine until they are removed.
-You can access this variable in your code like so:
-
-````{tabs}
+While we may not want to share files containing credentials, configuration and data, it can be useful to provide dummy examples of these files. Demonstrating the format of these files can help others to reuse your code, without sharing sensitive information.
 
-```{code-tab} py
-import os
+### Excluding `.Rdata` and `.Rhistory` files
```

Comment:
This should be nested under the gitignore heading. As its basically still a part of that